ICAFIX/scripts/icaDim.m

```diff
@@ -40,19 +45,25 @@
 [u,EigS,v]=nets_svds(data_detrend',0); %Eigenvalues of detrended data
 Out.DOF=sum(diag(EigS)>(std(diag(EigS))*0.1)); %Compute degrees of freedom
 
+clear data_detrend;
```

Comment:
clearing `data_detrend`, only to reconstitute later as a `(u*EigS*v')'`computation repeatedly within the `while` loop, seems a bit extreme.